README.md

```diff
@@ -76,3 +76,15 @@ datasources:
       accessKey: '<your access key>'
       secretKey: '<your secret key>'
 ```
+
+## How to build this datasource
+You will need to install [Node.js](https://nodejs.org/en/), [Yarn](https://yarnpkg.com/), [Go](https://golang.org/), and [Mage](https://magefile.org/) first.
+1. `yarn install --frozen-lockfile`
+1. `rm -rf node_modules/@grafana/data/node_modules`
+1. `npm run build`
```

Comment:
```suggestion
1. `yarn dev`
```
This will be the most forgiving way to build/run the environment

Reply:
Fixing in my latest commit. Please review them.